skr-api/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.71.0-slim-bullseye as build
+RUN apt-get update && apt-get install -y libssl-dev pkg-config protobuf-compiler curl gnupg2 build-essential clang
+RUN curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intel-sgx.list
+RUN apt-get update && apt-get install -y libtdx-attest-dev libtss2-dev
+COPY Cargo.* /build/
+COPY build.rs /build/
+COPY protos /build/protos
+COPY src /build/src
+WORKDIR /build
+RUN cargo b --release --bin skr-api
```

Comment:
Can we build a static binary?

Reply:
possibly, it's a bit of a headache with tpm libs, since they have an openssl dep, but we shouldn't need them.

Reply:
yes, see other comment.